runtime/handle.js

```diff
@@ -140,17 +140,24 @@ class Collection extends Handle {
   }
 
   // Called by StorageProxy.
```

Comment:
No longer true?

Reply:
Although it seems like it should be true - i.e. the scheduler should sit between handles and the particles so that all the changes for a particle can be aggregated and delivered in a single call?

Reply:
> Although it seems like it should be true - i.e. the scheduler should sit between handles and the particles so that all the changes for a particle can be aggregated and delivered in a single call?

I had thought about that a little, since event delivery can batch together events for different handles there's not an appropriate storage-proxy instance to call into. Instead I've been considering the scheduler to be an extension of the storage-proxy (so I've defined it in the same file as `StorageProxyScheduler`). If we need to schedule other things this might become a sub-component of a larger `Scheduler`.

Reply:
Let's talk about this face-to-face, I don't understand your comment sorry :)